package/MDAnalysis/exceptions.py

```diff
@@ -60,6 +60,10 @@ class FileFormatWarning(Warning):
     """Warning indicating possible problems with a file format."""
 
 
+class FileIOError(Exception):
```

Comment:
We don't want to introduce custom exceptions unless really necessary. (Originally we were quite generous with new warnings and exceptions but this tends to make things confusing for downstream code, which then has to import the custom exceptions to test against them etc.)

Unless there's a really good reason, please remove FileIOError.